agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -1697,12 +1697,13 @@ bool slave_thread::handle_cmdu_ap_manager_message(Socket *sd,
             supported_channels.begin(), &std::get<1>(tuple_supported_channels),
             &std::get<1>(tuple_supported_channels) + notification->supported_channels_size());
 
+        // If Monitor has already joined the stats has been incremented on the Monitor join handler,
```

Comment:
I don't understand this... Do you mean
```suggestion
        // If Monitor has already joined the state has been incremented on the Monitor join handler,
```
?

Reply:
yes, fixed.